addons/mail/static/src/components/thread_textual_typing_status/thread_textual_typing_status_tests.js

```diff
@@ -293,10 +314,13 @@ QUnit.test('receive several other members typing status "is typing"', async func
     // simulate receive typing notification from other11 (is typing)
     await afterNextRender(() => {
         const typingData = {
-            info: 'typing_status',
-            is_typing: true,
-            partner_id: 11,
-            partner_name: "Other11",
+            type: 'mail.typing_status',
+            payload: {
+                id: 20,
+                is_typing: true,
+                partner_id: 11,
+                partner_name: "Other11",
```

Comment:
note: so much diff for this typing stuff, we should probably have a test utils "simulate typing start" and "simulate typing end" with just the channel and partner as param.